agency-api-spec.md

```diff
@@ -175,35 +286,34 @@ JSON payload that contains the form fields.
 
 ```
 {
+    "version": "1.0.0",
+    "request_id": 1534,
     "agency": "Department of Justice",
     "agency_component_name": "Office of Information Policy",
-    "attachments": [
+    "attachments_supporting_documentation": [
         {
             "content_type": "application/pdf",
             "filedata": "YSBiYXNlNjQgZW5jb2RlZCBmaWxlCg==",
             "filename": "letter.pdf",
             "filesize": 27556
         }
     ],
-    "description": "I am seeking records pertaining to ...",
+    "request_description": "I am seeking records pertaining to ...",
     "email": "george.washington@example.com",
-    "expedited": false,
-    "fax": "+15551234589",
-    "fee_waiver": false,
-    "max_fee": 25.0,
-    "organization": "Newspaper Inc",
-    "phone": "+15551234567",
-    "requester_address": {
-        "address1": "1800 F Street",
-        "address2": "Suite 400",
-        "city": "Mount Vernon",
-        "state": "Virginia",
-        "zip": "98273"
-    },
-    "requester_name": {
-        "first": "George",
-        "last": "Washington"
-    }
+    "expedited_processing": "no",
+    "fax_number": "+15551234589",
+    "fee_waiver": "no",
+    "fee_willing_amount": 25.0,
+    "company_organization": "Newspaper Inc",
+    "phone_number": "+15551234567",
+    "address_line1": "1800 F Street",
+    "address_line2": "Suite 400",
+    "address_city": "Mount Vernon",
+    "address_state_province": "Virginia",
+    "address_country": "United States",
+    "address_zip_postal_code": "98273",
+    "name_first": "George",
+    "name_last": "Washington"
```

Comment:
Need to add the following to the sample payload and sample curl request:
- `request_category` (I noticed there is already a `request_origin` in the sample request, should that be replaced by this `request_category`?)
- `fee_waiver_explanation`
- `expedited_processing_explanation`